packages/core/content-manager/admin/src/pages/ListView/ListViewPage.tsx

```diff
@@ -203,7 +203,12 @@ const ListViewPage = () => {
     return <Page.Error />;
   }
 
-  const contentTypeTitle = schema?.info.displayName ?? 'Untitled';
+  const contentTypeTitle = schema?.info.displayName
+    ? formatMessage({ id: schema.info.displayName, defaultMessage: schema.info.displayName })
```

Comment:
This fixes the issue described here: https://github.com/strapi/strapi/issues/20461#issuecomment-2606591970